devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy

```diff
@@ -56,6 +61,12 @@ def parse(String description) {
 	if (result.name == "switch") {
 		result = createEvent(descriptionText: "Wake up event came in", isStateChange: true)
 	}
+    if ((device.getDataValue("model") == "Adurolight_NCC") || (device.getDataValue("model") == "ADUROLIGHT_CSC")) {
```

Comment:
it would be better to add this inside `parseAttrMessage` to maintain the parallel structure.